diff --git a/setup_dxvk.sh b/setup_dxvk.sh
index 0cd5d0a1..d257bd37 100644
--- a/setup_dxvk.sh
+++ b/setup_dxvk.sh
@@ -134,14 +134,4 @@ uninstall() {
   restoreDll "$1"
 }

-# skip dxgi during install if not explicitly
-# enabled, but always try to uninstall it
-if [ $with_dxgi -ne 0 ] || [ "$action" == "uninstall" ]; then
-  $action dxgi
-fi
-
 $action d3d9
-$action d3d10
-$action d3d10_1
-$action d3d10core
-$action d3d11
