diff --git a/src/phys_ctlr.cpp b/src/phys_ctlr.cpp
--- a/src/phys_ctlr.cpp
+++ b/src/phys_ctlr.cpp
@@ -430,20 +430,8 @@ enum phys_ctlr::PairingState phys_ctlr::get_pairing_state() const
                 state = PairingState::Virt_Procon;
             break;
         case Model::Left_Joycon:
-            if (l ^ zl)
-                state = PairingState::Waiting;
-            else if (sl && sr)
-                state = PairingState::Horizontal;
-            else if (l && zl)
-                state = PairingState::Lone;
-            break;
         case Model::Right_Joycon:
-            if (r ^ zr)
-                state = PairingState::Waiting;
-            else if (sl && sr)
-                state = PairingState::Horizontal;
-            else if (r && zr)
-                state = PairingState::Lone;
+            state = PairingState::Waiting;
             break;
         default:
             break;
