diff --git a/setup_dxvk.sh b/setup_dxvk.sh
index 01cebf7f..866e6455 100644
--- a/setup_dxvk.sh
+++ b/setup_dxvk.sh
@@ -50,7 +50,6 @@ if [ -z "$wine" ]; then
 fi

 wine64="${wine}64"
-wineboot="${wine}boot"

 # resolve 32-bit and 64-bit system32 path
 winever=`$wine --version | grep wine`
@@ -59,10 +58,6 @@ if [ -z "$winever" ]; then
     exit 1
 fi

-# ensure wine placeholder dlls are recreated
-# if they are missing
-$wineboot -u
-
 win32_sys_path=$($wine winepath -u 'C:\windows\system32' 2> /dev/null)
 win64_sys_path=$($wine64 winepath -u 'C:\windows\system32' 2> /dev/null)
